src/renderer/components/watch-video-comments/watch-video-comments.css

```
@@ -117,6 +117,14 @@
   height: 14px;
 }
 
+.commentSubscribedIcon {
+  content: var(--logo-icon);
+  height: 10px;
+  width: 10px;
```

As the font size of the author name is 14px, you could set this to 14px too, so that it's not so tiny. Additionally vertically aligning it, aligns it with the author name, giving it a slightly cleaner look.
```suggestion
  height: 14px;
  width: 14px;
  vertical-align: middle;
```